kube_metrics_server/datadog_checks/kube_metrics_server/kube_metrics_server.py

```diff
@@ -0,0 +1,105 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.checks.openmetrics import OpenMetricsBaseCheck
+
+COUNTERS = {
+    'authenticated_user_requests': 'authenticated_user.requests',
+    'metrics_server_kubelet_summary_scrapes_total': 'kubelet_summary_scrapes_total',
+    'process_cpu_seconds_total': 'process.cpu_seconds_total',
+}
+
+HISTOGRAMS = {
+    'metrics_server_manager_tick_duration_seconds': 'manager_tick_duration',
+    'metrics_server_scraper_duration_seconds': 'scraper_duration',
+}
+
+SUMMARIES = {'metrics_server_kubelet_summary_request_duration_seconds': 'kubelet_summary_request_duration'}
+
+GAUGES = {
+    'metrics_server_scraper_last_time_seconds': 'scraper_last_time',
+    'process_max_fds': 'process.max_fds',
+    'process_open_fds': 'process.open_fds',
+    'process_resident_memory_bytes': 'process.resident_memory_bytes',
+    'process_start_time_seconds': 'process.start_time_seconds',
+    'process_virtual_memory_bytes': 'process.virtual_memory_bytes',
+}
+
+GO_METRICS = {
```

Comment:
not sure we want to aggregate all the Go metrics in this check. (`kube-controller-manager` and `kube-scheduler`) it is not the case.
dd already have an integration for golang process.

Reply:
I see, do you suggest ignoring all GO_METRICS and keep `process.cpu_seconds_total` only ?

Reply:
or should we keep at least some metrics like `go.gc_duration_seconds` and `go.goroutines` ?